R/module-helpers.R

```diff
@@ -45,5 +45,35 @@ set_water_input_name <- function(water_sector, water_type, water_mapping, GLU =
            # irrigation mapped types - needs the GLU column
            new_name = if_else(water_sector == IRRIGATION & water_type %in% MAPPED_WATER_TYPES,
                               paste(supplysector, GLU, wt_short, sep = "_"), new_name)) %>%
-  .$new_name
+    .$new_name
+}
+
+
+#' rename_SO2
+#'
+#' Rename SO2 to regional SO2. (TODO)
+#'
+#' @param x Data object (typically from pipeline)
+#' @param so2_map TODO
+#' @param is_awb TODO
+#' @return Data object with \code{Non.CO2} changed to SO2 name for SO2 data.
+#' @importFrom tibble is_tibble
+#' @author BBL May 2017
```

Comment:
so2_map uses information from column  "SO2_name" in A_regions.csv, I believe.  The "is.awb" is about agriculture waste burning, and is also defined by the column "SO2_name" in A_regions.csv, I think.  I don't know how A_regions.csv is created or what all the SO2_names mean.